src/main/java/seedu/address/model/person/StaffInsurance.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a staff insurance in HR Pro Max++.
+ * Guarantees: immutable; is valid as declared in {@link #isValidName(String)}
+ */
+public class StaffInsurance {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Staff insurance should only contain alphanumeric characters and spaces, and it should not be blank";
+
+    /*
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    // TODO: 7/10/2022 check if this is the proper regex
```

Comment:
Remember to do this. Currently not sure what is the best validation for this. Need to add testscases to check.